src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx

```diff
@@ -34,6 +34,7 @@ export const ConsultationFeedTab = (props: ConsultationTabProps) => {
   const bed = props.consultationData.current_bed?.bed_object;
   const feedStateSessionKey = `encounterFeedState[${props.consultationId}]`;
   const [preset, setPreset] = useState<CameraPreset>();
+  const [selectedPreset, setSelectedPreset] = useState<CameraPreset>();
```

Comment:
this is now confusing... so what does `preset` state store?

Reply:
> this is now confusing... so what does `preset` state store?

Specifically stores user's explicit selection, whereas preset is tied to the asset object.

https://github.com/ohcnetwork/care_fe/blob/66326dd053cffbb7020f4381ad397217b4de2698/src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx#L57